graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DataLoaderProvider.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.internal
```

Comment:
Looks like you are expecting the `DataLoaderProvider` to be a _none internal_ interface such that developers can provide their own?

Reply:
Yes, it meant to be publicly available
Could you suggest the right package?

Reply:
sure, `com.netflix.graphql.dgs`

Reply:
We should probably make this `DgsDataLoaderProvider` and our current implementation be `DefaultDgsDataLoaderProvider`